course/src/main/java/in/testpress/course/util/EncryptDecrypt.kt

```diff
@@ -0,0 +1,106 @@
+package `in`.testpress.course.util
+
+import android.content.Context
+import android.util.Base64
+import java.io.*
+import java.security.SecureRandom
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.spec.SecretKeySpec
+
+class FileEncryptionAndDecryption(val context: Context) {
+
+    private var sharedPref = context.applicationContext.getSharedPreferences("secretKeyPref", 0)
+
+    fun encryptDownloadedFile(filePath: String) {
```

Comment:
Does it matter if it is a downloaded file or non-downloaded file ? Encrypt method should encrypt any file.